src/qibosoq/qick_programs.py

```diff
@@ -110,6 +118,7 @@ def __init__(
 
         # general settings
         self.max_gain = qpcfg.max_gain
+        self.max_flux_gain = 24000  # TODO not hardcode
```

Comment:
I agree with the comment: in order not to forget (the `TODO` is already good in this sense), a better temporary solution is to set this variable as a global constant, and use the constant instead of the literal.